packages/manager/modules/overthebox/package.json

```
@@ -16,11 +16,11 @@
     "lodash": "^4.17.15"
   },
   "devDependencies": {
-    "@ovh-ux/component-rollup-config": "^7.0.0 || ^8.0.0"
+    "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
     "@ovh-ux/manager-core": "^11.0.0 || ^12.0.0",
+    "@ovh-ux/manager-ng-layout-helpers": "^1.0.0 || ^2.0.0",
```

```suggestion
    "@ovh-ux/manager-ng-layout-helpers": "^2.0.0",
```
I think as package has now been released